vendor/ddcloud/resource_virtual_listener.go

```diff
@@ -226,6 +227,7 @@ func resourceVirtualListenerCreate(data *schema.ResourceData, provider interface
 		return fmt.Errorf("Cannot find newly-created virtual listener with Id '%s'.", virtualListenerID)
 	}
 
+	data.Set(resourceKeyVirtualListenerIPv4Address, virtualListener.ListenerIPAddress)
 	// TODO: Populate computed properties.
```

Comment:
Since we are now populating computed properties, would you remind removing this comment, please?